pr2eus/robot-interface.l

```diff
@@ -246,10 +246,10 @@
          (let ((max-time (apply #'max time-list)))
            (max max-time min-time))))))
   (:angle-vector
-   (av &optional (tm nil) (ctype controller-type) (start-time 0) &key (scale 5) (min-time 1.0))
+   (av &optional (tm nil) (ctype controller-type) (start-time 0) &key (scale 1) (min-time 1.0))
    "tm (time to goal in msec)
    if designated tm is faster than fastest speed, use fastest speed
-   if not specified, it will use 1/5 of the fastest speed .
+   if not specified, it will use 1/scale of the fastest speed .
    if :fastest is specefied use fastest speed calcurated from max speed"
```

Comment:
if not specifined, that is max speed, so we do not need :fastest anymore, is it correct?

https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/86#issuecomment-83123289